Lebenslauf.html

```diff
@@ -1,142 +1,152 @@
-      <th>EDV</th>
-      <th>Kenntnisstand</th>
-    </tr>
-   <tr>
-     <td>MS Word 2010</td>
-     <td>Gute Kenntnisse</td>
-   </tr>
-   <tr>
-     <td>MS Excel 2010</td>
-     <td>Grundkenntnisse</td>
-   </tr>
-   <tr>
-     <td>MS Powerpoint 2010</td>
-     <td>Grundkenntnisse</td>
-   </tr>
-   <tr>
-     <td>Markdown</td>
-     <td>Grundkenntnisse</td>
-    </tr>
+
+    <h1 style="font-size:200%; font-family:verdana;">Lebenslauf</h1>
+    <p>Luise Maron</p>
+    <img src="Passfoto.jpg" width="150" height="224">
+    <p>Lennéstraße 19<br> 14469 Potsdam</p>
+    <p>E-Mail: luisemaron@googlemail.com</p>
+    <p>Geburtsort &amp; -datum: Potsdam, den 07. April 1989</p>
+    <p>Staatsangehörigkeit: deutsch</p>
+    <p>Familienstand: ledig</p>
+    <h2 style="color:blue; font-family:verdana;">Berufserfahrung</h2>
+    <hr>
+
+    <table>
+        <tr>
+            <th style="text-align:center;">Zeitraum</th>
+            <th style="text-align:center;">Tätigkeit</th>
+        </tr>
+        <tr>
+            <td><i>2014-heute</i></td>
+            <td>Aushilfskraft bei <a href="http://www.hugendubel.de/de/">Hugendubel</a></td>
+            <tr>
+                <td></td>
+                <td>Verkauf von Büchern und anderen Medien, Kundenberatung und -bedienung, Bibliographieren und Recherchieren, Beschaffung von Kundenbestellungen, Sortimentsgestaltung</td>
+            </tr>
+            <tr>
+                <td><i>2010-2012</i></td>
+                <td>Aushilfskraft bei Eismanufaktur Potsdam</td>
+            </tr>
+            <tr>
+                <td></td>
+                <td>Tresen- und Servicekraft</td>
+            </tr>
+            <tr>
+                <td><i>2007-2014</i></td>
+                <td>Hostess &amp; Chefhostess bei Heiden &amp; Wo-Men Veranstaltungsservice</td>
+            </tr>
+            <tr>
+                <td></td>
+                <td>Delegation von Teams mit mehr als fünf Mitarbeitern, Koordination von Veranstaltungsabläufen, Organisation und Nachbereitung vor Ort</td>
+            </tr>
+    </table>
+
+    <h2 style="color:blue; font-family:verdana;">Ausbildung</h2>
+    <hr>
+
+    <table>
+        <tr>
+            <th>Zeitraum</th>
+            <th>Tätigkeit</th>
+        </tr>
+        <tr>
+            <td>2010-2015</td>
+            <td>B.A. Germanistik und Spanische Philologie, <a href="https://www.uni-potsdam.de/">Universität Potsdam</a></td>
+        </tr>
+        <tr>
+            <td>2009-2010</td>
+            <td>Spanische Philologie und Linguistik, <a href="https://www.uni-potsdam.de/">Universität Potsdam</a></td>
+        </tr>
+        <tr>
+            <td>2008-2009</td>
+            <td>Französische Philologie und Linguistik, <a href="https://www.uni-potsdam.de/">Universität Potsdam</a></td>
+        </tr>
+        <tr>
+            <td>2001-2008</td>
+            <td>Allgemeine Hochschulreife, <a href="https://www.wolkenberg-gymnasium.de/">Wolkenberg Gymnasium Michendorf</a></td>
```

Comment:
If you don't want people to leave your CV when they click the link, but rather want to page to open in a new tab, you should have a look at the [`target`](http://www.w3schools.com/tags/att_a_target.asp)-attribute.